nvflare/app_common/hub/hub_controller.py

```diff
@@ -348,39 +350,45 @@ def _control_flow(self, abort_signal: Signal, fl_ctx: FLContext):
                     )
                     return
             else:
-                if topic in [Topic.ABORT, Topic.END, Topic.PEER_GONE]:
+                assert isinstance(msg, Message)
```

Comment:
note that assert will not work if start python with "-O" options (which should be the case for production use)

Reply:
Removed as suggested